js/src/forum/fragments/PostQuoteButton.js

```diff
@@ -0,0 +1,54 @@
+import Fragment from 'flarum/Fragment';
+import icon from 'flarum/helpers/icon';
+
+import reply from '../utils/reply';
+
+export default class PostQuoteButton extends Fragment {
+  constructor(post) {
+    super();
+
+    this.post = post;
+  }
+
+  view() {
+    return (
+      <button class="Button PostQuoteButton" onclick={() => {
+        this.hide();
+        reply(this.post, this.content);
+      }} mousedown={(e) => e.stopPropagation()}>
```

Comment:
This would need to be `mouseup` now as well.

Reply:
Even better, that code doesn't actually run, so we should be good just removing it.

Reply:
Should it have been **on**mousedown?

Reply:
Oh no! :facepalm: Yeah, that would explain the related bugs we saw, and why the mousedown handler elsewhere needed to be changed to mouseup. That being said, the way it is now works, and is a bit simpler IMO (it'll ALWAYS hide the button on mouseup, and if we clicked on it in the process, it'll also execute a new reply